feat(content): Add a mission transporting a human to Hai space (#5927)
endless-sky/endless-sky: data/human/transport missions.txt

			label reject
			`	Charles looks surprised for a moment, and then, for the first time you've seen, he smiles. "Thank you, Captain <last>. I wish there were more people like you in the Paradise Worlds."`
			`	You wish the couple the best of luck on Earth, and they slowly make their way to the train station. Presumably, their new apartment is far from here.`

mission "Human to Hai Space"
	description "Bring a human passenger to <destination> in Hai space."
	minor
	source
		government "Republic" "Syndicate" "Free Worlds"
		attributes "tourism" "urban"
	destination 
		government "Hai"
		attributes "human tourism"
	passengers 1
	to offer
		random < 10
	on offer
		conversation
			`A young person approaches you carrying luggage. "You look like a knowledgeable captain," he says. "Do you know of a place I can go to visit some... intelligent rodents?"`
			branch "hai contact"
				has "First Contact: Hai: declined"
			
			choice

> 				`   "What are you talking about?"`

Author: I believe I've fixed it, but I'm not sure. Is it alright now?

				`   "What, like in a zoo?"`
					goto "confused"
				`   "Are you high?"`
			`   "No, I want to go to the Hai-" He stops once he realizes you have no idea what he's talking about. "Oh, never mind then," he says, with a note of disappointment in his voice. As he leaves, you can't help feeling like you were missing some context.`
				decline
			label "confused"
			`   "Oh, never mind then," he says, with a note of disappointment in his voice. As he leaves, you can't help feeling like you were missing some context.`
				decline
			
			label "hai contact"
			choice
				`   "You mean the Hai? You'll have to go up north for that."`
					goto "squirrel transport"
				`   "Ah, I think I know of some large squirrels to the north of here."`
					goto "squirrel transport"

> 				`   "What are you talking about?"`
> 			`	"Oh, never mind then," he says, with a note of disappointment in his voice. As he leaves, you can't help feeling like you were missing some context.`

Reviewer: This line is missing a decline and will run right into the next label as if you had told him where to go, but really you should just wrap back up to the line that already declines instead of duplicating the dialog.
```suggestion
				`	"What are you talking about?"`
					goto confused
```

			label "squirrel transport"
			`   "If that's the case, could you bring me to <destination>? I'll give you <payment>."`
			choice
				`   "Alright, I'll take you there."`
					accept
				`   "Sorry, I'm too busy right now."`
					decline

> 				`   "Why would you want to go all the way out to <destination>?"`

Reviewer 2: `<destination>` is replaced by `<planet> in the <system> system`. Having it in both the question and the answer feels weird:
```
- "If that's the case, could you bring me to Greenwater in the Fah Soom system? I'll give you <payment>."
- "Why would you want to go all the way out to Greenwater in the Fah Soom system?"
```

Using only \<planet\> or \<system\> in the answer would be better:
```
- "If that's the case, could you bring me to Greenwater in the Fah Soom system? I'll give you <payment>."
- "Why would you want to go all the way out to Greenwater?"
```

			`   "I have family there," he says. "I came out here to see a bit of human space, but now I have to get back home."`
			choice
				`   "Alright, I'll take you there."`
					accept
				`   "Sorry, I'm too busy right now."`
					decline
	on complete

> 		payment 8000 1000

Reviewer: You need to have a fixed payment for this mission. Payment per jump doesn't work through wormholes.

            branch "hai contact"
                has "First Contact: Hai: declined"
            
            choice
                `   "What are you talking about?"`
                `   "What, like in a zoo?"`
                    goto "confused"
                `   "Are you high?"`
                    goto "bad pun"
            label "confused"
            `   "Oh, never mind then," he says, and he leaves. You can't help feeling like you were missing some context.`
                decline
            label "bad pun"
            `   "No, I want to go to the Hai-" He stops once he realizes you have no idea what he's talking about. "Oh, never mind then," he says, and he leaves. You can't help feeling like you were missing some context.`
                decline
            
            label "hai contact"
            choice
                `   "You mean the Hai? You'll have to go up north for that."`
                    goto "squirrel transport"
                `   "Ah, I think I know of some large squirrels to the north of here."`
                    goto "squirrel transport"

>                 `   (Feign ignorance.)`

Reviewer 3: I think it would be better if you brought the "I don't know what you're talking about" here for consistency

Author: Do you mean that I should replace `(Feign ignorance.)` with `I don't know what you're talking about.`?

Reviewer 3: Yeah

                    goto "ignorance"
            label "ignorance"
            `   "I have no idea what you're talking about," you say. "Oh, never mind then," he responds, and he leaves.`
            label "squirrel transport"
            `   "If that's the case, could you bring me to <destination>? I'll give you <payment>."`
            choice
                `   "Alright, I'll take you there."`
                    accept
                `   "Sorry, I'm too busy right now."`
                    decline

>                 `   "Why would you want to go all the way out to <destination>?"`

Reviewer 2: `<destination>` is replaced by `<planet> in the <system> system`. Having it in both the question and the answer feels weird:
```
- "If that's the case, could you bring me to Greenwater in the Fah Soom system? I'll give you <payment>."
- "Why would you want to go all the way out to Greenwater in the Fah Soom system?"
```

Using only \<planet\> or \<system\> in the answer would be better:
```
- "If that's the case, could you bring me to Greenwater in the Fah Soom system? I'll give you <payment>."
- "Why would you want to go all the way out to Greenwater?"
```

				decline
			label "confused"
			`	"Oh, never mind then," he says, with a note of disappointment in his voice. As he leaves, you can't help feeling like you were missing some context.`
				decline
			
			label "hai contact"
			choice
				`	"You mean the Hai? You'll have to go up north for that."`
					goto "squirrel transport"
				`	"Ah, I think I know of some large squirrels to the north of here."`
					goto "squirrel transport"
				`	"What are you talking about?"`
					goto "confused"
			label "squirrel transport"
			`	"If that's the case, could you bring me to <destination>? I'll give you <payment>."`
			choice
				`	"Alright, I'll take you there."`
					accept
				`	"Sorry, I'm too busy right now."`
					decline
				`	"Why would you want to go all the way out to <planet>?"`
			`	"I have family there," he says. "I came out here to do some traveling and see a bit of human space, but now I have to get back home. You see, I was born on <planet>, and this is my first time out of the wormhole."`
			choice
				`	"Really? How did you like human space?"`
					goto "travel"
				`	"You've never been to human space before?"`
					goto "space"
			label "travel"

> 				`	"It was quite the trip! It's a bit odd to see only humans around, and I had to be careful to not mention where I was from. But it was definitely worth it! I got to see Earth, Skymoot, Chiron, and so many other beautiful planets. Seeing human culture with my own eyes was an eye-opening experience, although some of the places there were less... clean than I expected. I was almost robbed on Earth; I'm not used to pickpockets back home. Perhaps human space isn't as safe as Hai space, but I still don't regret coming here. It's about time for me to go back, though."`

Reviewer 2: Currently the "travel" part continues into the "space" part. I think it should go straight to the next choice.

Reviewer 2: Well, now it's the other way around. In game, you'll get: 
"So anyways, would you be willing to take me to <destination>?" 
and then, instead of letting you answer, he'll continue with the "travel" part.

Reviewer 2: Unrelated to my previous comments: there is an extra tab.
```suggestion
			`	"It was quite the trip! It's a bit odd to see only humans around, and I had to be careful to not mention where I was from. But it was definitely worth it! I got to see Earth, Skymoot, Chiron, and so many other beautiful planets. Seeing human culture with my own eyes was an eye-opening experience, although some of the places there were less... clean than I expected. I was almost robbed on Earth; I'm not used to pickpockets back home. Perhaps human space isn't as safe as Hai space, but I still don't regret coming here. It's about time for me to go back, though."`
```

			label "space"
				`	"Nope, first time! I've never been so far from home before. Now that I think about it, I probably should've arranged transport back to Hai space beforehand. You wouldn't believe how hard it is to find a captain here that knows about the Hai. I had to ask about six others until I found you. They all thought I was crazy, talking about meeting giant squirrels." He laughs and shakes his head. "So anyways, would you be willing to take me to <destination>?"`
			choice
				`	"Alright, I'll take you there."`
					accept
				`	"Sorry, I'm too busy right now."`
					decline
	on complete
		payment 9000
		conversation
			`During the trip, you learn that your passenger is named Benjamin. He gets off your ship, thanks you and gives you your payment of <payment>. Benjamin then walks over to greet a group of Hai, who embrace him and start talking in a mix of the human and Hai languages.`
			choice
				`	(Leave.)`
					decline
				`	"That's your family?"`

> 			`	The man looks at you. "I'm adopted," he says. One of the Hai notices your confusion, and chitters. "Ah, you have not seen a human-Hai family, have you? Perhaps it is strange, but we are happy together nonetheless. Thank you for transporting our som safely."`

Reviewer 2: ```suggestion
			`	The man looks at you. "I'm adopted," he says. One of the Hai notices your confusion, and chitters. "Ah, you have not seen a human-Hai family, have you? Perhaps it is strange, but we are happy together nonetheless. Thank you for transporting our son safely."`
```

				decline
			label "confused"
			`	"Oh, never mind then," he says, with a note of disappointment in his voice. As he leaves, you can't help feeling like you were missing some context.`
				decline
			
			label "hai contact"
			choice
				`	"You mean the Hai? You'll have to go up north for that."`
					goto "squirrel transport"
				`	"Ah, I think I know of some large squirrels to the north of here."`
					goto "squirrel transport"
				`	"What are you talking about?"`
					goto "confused"
			label "squirrel transport"
			`	"If that's the case, could you bring me to <destination>? I'll give you <payment>."`
			choice
				`	"Alright, I'll take you there."`
					accept
				`	"Sorry, I'm too busy right now."`
					decline
				`	"Why would you want to go all the way out to <planet>?"`
			`	"I have family there," he says. "I came out here to do some traveling and see a bit of human space, but now I have to get back home. You see, I was born on <planet>, and this is my first time out of the wormhole."`
			choice
				`	"Really? How did you like human space?"`
					goto "travel"
				`	"You've never been to human space before?"`

> 			`	"Nope, first time! I've never been so far from home before. Now that I think about it, I probably should've arranged transport back to Hai space beforehand. You wouldn't believe how hard it is to find a captain here that knows about the Hai. I had to ask about six others until I found you. They all thought I was crazy, talking about meeting giant squirrels." He laughs and shakes his head. "So anyways, would you be willing to take me to <destination>?"`

Reviewer 2: ```suggestion
			`	"Nope, first time! I've never been so far from home before. Now that I think about it, I probably should've arranged transport back to Hai space beforehand. You wouldn't believe how hard it is to find a captain here that knows about the Hai. I had to ask about six others until I found you. They all thought I was crazy, talking about meeting giant squirrels." He laughs and shakes his head. "So anyways, would you be willing to take me to <destination>?"`
				goto "decision"
```
To be added along with a 'label "decision"' right before the next choice.

Reviewer 4: ```suggestion
			`	"Nope, first time! I've never been so far from home before. Now that I think about it, I probably should've arranged transport back to Hai space beforehand. You wouldn't believe how hard it is to find a captain here who knows about the Hai. I had to ask about six others until I found you. They all thought I was crazy, talking about meeting giant squirrels." He laughs and shakes his head. "So anyways, would you be willing to take me to <destination>?"`
```

			label "travel"
				`	"It was quite the trip! It's a bit odd to see only humans around, and I had to be careful to not mention where I was from. But it was definitely worth it! I got to see Earth, Skymoot, Chiron, and so many other beautiful planets. Seeing human culture with my own eyes was an eye-opening experience, although some of the places there were less... clean than I expected. I was almost robbed on Earth; I'm not used to pickpockets back home. Perhaps human space isn't as safe as Hai space, but I still don't regret coming here. It's about time for me to go back, though."`
				

> 			choice

Reviewer 2: ```suggestion
			label "decision"
			choice
```

				`	"Alright, I'll take you there."`
					accept
				`	"Sorry, I'm too busy right now."`
					decline
	on complete
		payment 9000
		conversation
			`During the trip, you learn that your passenger is named Benjamin. He gets off your ship, thanks you and gives you your payment of <payment>. Benjamin then walks over to greet a group of Hai, who embrace him and start talking in a mix of the human and Hai languages.`
			choice
				`	(Leave.)`
					decline
				`	"That's your family?"`
			`	The man looks at you. "I'm adopted," he says. One of the Hai notices your confusion, and chitters. "Ah, you have not seen a human-Hai family, have you? Perhaps it is strange, but we are happy together nonetheless. Thank you for transporting our son safely."`
			choice
				`	"I didn't mean to be rude, I just didn't know that Hai adopted humans here."`
				`	"No problem, I hope you enjoyed the trip."`

> 			`After thanking you once again, they walk off. As they leave, you overhear Benjamin talk to his family about his trip. "They had so many kinds of food there, it was delicious! Although I did get homesick for pickled acorns..."`

Reviewer 2: ```suggestion
			`	After thanking you once again, they walk off. As they leave, you overhear Benjamin talk to his family about his trip. "They had so many kinds of food there, it was delicious! Although I did get homesick for pickled acorns..."`
```

				
			label "decision"
			choice
				`	"Alright, I'll take you there."`
					accept
				`	"Sorry, I'm too busy right now."`
					decline
	on complete
		payment 9000
		conversation
			`During the trip, you learn that your passenger is named Benjamin. He gets off your ship, thanks you and gives you your payment of <payment>. Benjamin then walks over to greet a group of Hai, who embrace him and start talking in a mix of the human and Hai languages.`
			choice
				`	(Leave.)`
					decline
				`	"That's your family?"`
			`	The man looks at you. "I'm adopted," he says. One of the Hai notices your confusion, and chitters. "Ah, you have not seen a human-Hai family, have you? Perhaps it is strange, but we are happy together nonetheless. Thank you for transporting our son safely."`
			choice
				`	"I didn't mean to be rude, I just didn't know that Hai adopted humans here."`
				`	"No problem, I hope you enjoyed the trip."`
			`	After thanking you once again, they walk off. As they leave, you overhear Benjamin talk to his family about his trip. "They had so many kinds of food there, it was delicious! Although I did get homesick for pickled acorns..."`

> 				decline

Reviewer 5: ```suggestion
```

				`   "What, like in a zoo?"`
					goto "confused"
				`   "Are you high?"`
			`   "No, I want to go to the Hai-" He stops once he realizes you have no idea what he's talking about. "Oh, never mind then," he says, with a note of disappointment in his voice. As he leaves, you can't help feeling like you were missing some context.`
				decline
			label "confused"

> 			`   "Oh, never mind then," "Oh, never mind then," he says, with a note of disappointment in his voice. As he leaves, you can't help feeling like you were missing some context.`

Reviewer 2: You added an extra "Oh, never mind then,"

	on complete
		payment 30000
		dialog "As you descend through the upper atmosphere, you inform your skydivers that it's time to jump. However, after being able to see the distances and atmospheric conditions involved, all of them are far less keen on the idea than when you first picked them up. You bring them down to the landing pad, and they pay you a meager <payment> before hopping off."

mission "Human to Hai Space - No Contact"
	minor
	source
		government "Republic" "Syndicate" "Free Worlds"
		attributes "tourism" "urban"
	to offer
		random < 2
		not "First Contact: Hai: offered"
	on offer
		conversation

> 			`A young person approaches you carrying luggage. "You look like a knowledgeable captain," he says. "Do you know of a place I can go to visit some... intelligent rodents?"`

Reviewer 4: I might recommend having this fellow start with a more furtive opening, like he's trying to test the waters.

Something more like this:
```suggestion
			`A young man carrying luggage approaches you. "You look like a knowledgeable captain," he says. "In all your travels, have you ever heard tell of any... intelligent rodents?"``
```

			choice
				`	"What are you talking about?"`
					goto confused
				`	"What, like in a zoo?"`
					goto confused
				`	"Are you high?"`
			`	"No, I want to go to the Hai-" He stops once he realizes you have no idea what he's talking about. "Oh, never mind then," he says, with a note of disappointment in his voice. As he leaves, you can't help feeling like you were missing some context.`
				decline
			label confused
			`	"Oh, never mind then," he says, with a note of disappointment in his voice. As he leaves, you can't help feeling like you were missing some context.`
				decline

> mission "Human to Hai Space"

Reviewer 6: Just playtested - this mission needs a destination!

Author: Oops, fixed!

	minor
	description "Bring a human passenger to <destination> in Hai space."
	source
		government "Republic" "Syndicate" "Free Worlds"
		attributes "tourism" "urban"
	to offer
		random < 10
		has "First Contact: Hai: offered"
		not "Human to Hai Space - No Contact: offered"
	on offer
		conversation

> 			`A young person approaches you carrying luggage. "You look like a knowledgeable captain," he says. "Do you know of a place I can go to visit some... intelligent rodents?"`

Reviewer 4: (ditto for this version of the mission)

			choice
				`	"You mean the Hai? You'll have to go up north for that."`
					goto hai
				`	"Ah, I think I know of some large squirrels to the north of here."`
					goto hai
				`	"What are you talking about?"`

> 			`	"Oh, never mind then," he says, with a note of disappointment in his voice. As he leaves, you can't help feeling like you were missing some context.`

Reviewer 4: In this case you're not actually missing any context at all; you're choosing to play dumb on purpose!

```suggestion
			`	"Oh, never mind then," he says, with a note of disappointment in his voice. You feel good knowing that you are helping to keep the Hai's existence a secret in Human space. Better safe than sorry.`
```

Author: Thanks for catching this, that would've been embarrassing if it got merged.

				goto decision
			
			label travel
			`	"It was quite the trip! It's a bit odd to see only humans around, and I had to be careful to not mention where I was from. But it was definitely worth it! I got to see Earth, Skymoot, Chiron, and so many other beautiful planets. Seeing human culture with my own eyes was an eye-opening experience, although some of the places there were less... clean than I expected. I was almost robbed on Earth; I'm not used to pickpockets back home. Perhaps human space isn't as safe as Hai space, but I still don't regret coming here. It's about time for me to go back, though."`
				
			label decision
			choice
				`	"Alright, I'll take you there."`
					accept
				`	"Sorry, I'm too busy right now."`
					decline
	on complete
		payment 9000
		conversation

> 			`During the trip, you learn that your passenger is named Benjamin. He gets off your ship, thanks you and gives you your payment of <payment>. Benjamin then walks over to greet a group of Hai, who embrace him and start talking in a mix of the human and Hai languages.`

Reviewer 4: ```suggestion
			`During the trip, you learn that your passenger is named Benjamin. He gets off your ship, thanks you, and gives you your payment of <payment>. Benjamin then walks over to greet a group of Hai, who embrace him and start talking in a mix of the human and Hai languages.`
```

	on complete
		payment 30000
		dialog "As you descend through the upper atmosphere, you inform your skydivers that it's time to jump. However, after being able to see the distances and atmospheric conditions involved, all of them are far less keen on the idea than when you first picked them up. You bring them down to the landing pad, and they pay you a meager <payment> before hopping off."

mission "Human to Hai Space - No Contact"
	minor
	source
		government "Republic" "Syndicate" "Free Worlds"
		attributes "tourism" "urban"
	to offer
		random < 2
		not "First Contact: Hai: offered"
	on offer
		conversation
			`	A young man carrying luggage approaches you. "You look like a knowledgeable captain," he says. "In all your travels, have you ever heard tell of any... intelligent rodents?"`
			choice
				`	"What are you talking about?"`
					goto confused
				`	"What, like in a zoo?"`
					goto confused
				`	"Are you high?"`

> 			`	"No, I want to go to the Hai-" He stops once he realizes you have no idea what he's talking about. "Oh, never mind then," he says, with a note of disappointment in his voice. As he leaves, you can't help feeling like you were missing some context.`

Reviewer 4: since "Hai" and "high" are honophones, I think it would be funnier and less spoilery if you changed it around to use the other word:

```suggestion
			`	"No, I want to go to the high-" He stops once he realizes you have no idea what he's talking about. "Oh, never mind then," he says, with a note of disappointment in his voice. As he leaves, you can't help feeling like you were missing some context.`
```
